tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -323,4 +324,34 @@ public function userGetsTheInformationOfTheLastOpenedFileUsingWopiEndpoint(strin
 			)
 		);
 	}
+
+	/**
+	 * @Then /^the response should (not|)\s?contain following MIME Type:$/
+	 *
+	 * @param string    $shouldOrNot   (not|)
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theFollowingMimeTypesShouldExistForUser(string $shouldOrNot, TableNode $table):void {
+		$rows = $table->getRows();
+		foreach ($rows as $row) {
+			$responseArray = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())['mime-types'];
+			$mimeTypes = \array_column($responseArray, 'mime_type');
```

Comment:
these can be done once
```suggestion
		$rows = $table->getRows();
			$responseArray = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())['mime-types'];
			$mimeTypes = \array_column($responseArray, 'mime_type');
		foreach ($rows as $row) {
```